includes/Admin/Abstract_Settings_Screen.php

```diff
@@ -83,14 +83,33 @@ public function render() {
 	 *
 	 * @since 2.0.0
 	 *
-	 * @throws SV_WC_Plugin_Exception
+	 * @throws Framework\SV_WC_Plugin_Exception
 	 */
 	public function save() {
 
 		woocommerce_update_options( $this->get_settings() );
 	}
 
 
+	/**
+	 * Determines whether the current screen is the same as identified by the current class.
+	 *
+	 * @since 2.2.0-dev.1
+	 *
+	 * @return bool
+	 */
+	protected function is_current_screen_page() {
+
+		if ( Settings::PAGE_ID !== Framework\SV_WC_Helper::get_requested_value( 'page' ) ) {
+			return false;
+		}
+
+		$tab = Framework\SV_WC_Helper::get_requested_value( 'tab' );
```

Comment:
We need to offer a default here, because the link under the Marketing menu doesn't specify a `tab` parameter:

```suggestion
		// assume we are on the Connection tab by default because the link under Marketing doesn't include the tab query arg
		$tab = Framework\SV_WC_Helper::get_requested_value( 'tab', 'connection' );
```

Without the default the CSS stylesheet is not enqueued when you go to the Facebook for WooCommerce settings page clicking Marketing > Facebook.